bundles/TinymceBundle/public/js/editor.js

```diff
@@ -121,7 +122,13 @@ pimcore.bundle.tinymce.editor = Class.create({
                         }
                     }));
                 }.bind(this));
+                editor.on('change', function (eChange) {
+                    changedContent = true;
+                }.bind(this));
                 editor.on('blur', function (eChange) {
+                    if (!changedContent) {
+                        return;
+                    }
```

Comment:
Should the `changedContent` variable be set back to false so the event is only triggered when it is changed again?
```suggestion
                    }
                    changedContent = false;
```

Reply:
Thanks @blankse  Logically it makes sense to put it there as it will not trigger additional event pimcore.events.changeWysiwyg when content is not changed. I will add new commit.

Reply:
Thanks @blankse  Logically it makes sense to put it there as it will not trigger additional event pimcore.events.changeWysiwyg when content is not changed. I will add new commit.

Reply:
I added new commit